test/integration_test_helper.rb

```diff
@@ -20,18 +20,40 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
+
+options = Selenium::WebDriver::Chrome::Options.new
+options.args << '--disable-gpu'
+options.args << '--no-sandbox'
+options.args << '--window-size=1024,768'
+options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
+
+selenium_remote_host = ENV.fetch('SELENIUM_REMOTE_HOST', nil)
+selenium_remote_port = ENV.fetch('SELENIUM_REMOTE_PORT', 4444)
+
 Capybara.register_driver :selenium_chrome do |app|
-  options = Selenium::WebDriver::Chrome::Options.new
-  options.args << '--disable-gpu'
-  options.args << '--no-sandbox'
-  options.args << '--window-size=1024,768'
-  options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
-  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+  if selenium_remote_host
+    Capybara::Selenium::Driver.new(
```

Comment:
I will register a new symbol called `:selenium_chrome_remote`

The remote driver requires the options to be set:
```
msg: ArgumentError: Selenium::WebDriver::Remote::Driver needs :options to be set
```

But this can be solved by just passing the initialized object. Depending on the discussion from above if all options can be removed.